core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -100,13 +106,23 @@ object ZkUtils {
     (zkClient, zkConnection)
   }
 
-  def DefaultAcls(isSecure: Boolean): java.util.List[ACL] = if (isSecure) {
-    val list = new java.util.ArrayList[ACL]
-    list.addAll(ZooDefs.Ids.CREATOR_ALL_ACL)
-    list.addAll(ZooDefs.Ids.READ_ACL_UNSAFE)
-    list
-  } else {
-    ZooDefs.Ids.OPEN_ACL_UNSAFE
+  def SensitivePath(path: String): Boolean = {
+    path == null || !SensitiveZkRootPaths.forall(!path.startsWith(_))
+  }
+
+  @deprecated("This is deprecated, use DefaultAcls(isSecure, path) which doesn't make sensitive data world readable", since = "0.11.0.0")
```

Comment:
Should it not be 0.10.2.1 since you set the target version of the JIRA to 0.10.2.1?